contracts/mvx-game-sc/tests/game_sc_blackbox_tests.rs

```diff
@@ -102,15 +139,15 @@ impl GameContractState {
         waiting_time: u64,
         number_of_players_min: u64,
         number_of_players_max: u64,
-        wager: BigUint,
+        wager: BigUint<StaticApi>,
         caller: &str,
         game_sc: &mut ContractInfo<mvx_game_sc::Proxy<StaticApi>>,
     ) -> &mut Self {
         self.world.sc_call(
             ScCallStep::new()
                 .from(caller)
                 .to(GAME_SC_ADDR)
-                .esdt_transfer(TOKEN_ID, 0u64, BigUint::from(STARTING_FEE))
+                .esdt_transfer(TOKEN_ID, 0u64, BigUint::<StaticApi>::from(STARTING_FEE))
```

Comment:
I thinkg `BigUint::from(STARTING_FEE)` should be enough. Check for the imported BigUint (because there are 2).
You should use `multiversx_sc_scenario::num_bigint::BigUint`.